sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/entries/CreateNewTravelEntrySteps.java

```diff
@@ -222,6 +222,7 @@ public CreateNewTravelEntrySteps(
     When(
         "I check if data in case based on travel entry is correct",
         () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(By.id("info"));
```

Comment:
move it to locators class

Reply:
:D sorry, was checking it if it helps in stabilisation and forgot about it